pom.xml

```diff
@@ -209,15 +211,19 @@
         <libpam4j.version>1.11</libpam4j.version>
         <log4j.version>1.2.17</log4j.version>
         <log4j2.version>2.12.1</log4j2.version>
+        <lombok-version>1.18.4</lombok-version>
+        <log4j2.version>2.12.1</log4j2.version>
         <maven-checkstyle-plugin.version>3.1.0</maven-checkstyle-plugin.version>
         <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
         <maven-dependency-plugin.version>3.1.1</maven-dependency-plugin.version>
         <maven-enforcer-plugin.version>3.0.0-M2</maven-enforcer-plugin.version>
         <maven-pmd-plugin.version>3.12.0</maven-pmd-plugin.version>
         <metrics.version>4.1.0</metrics.version>
+        <mockito.version>1.10.19</mockito.version>
```

Comment:
Do we need `mockito`? We use `easymock` everywhere else for mocking. I don't have anything really against `mockito` we just tried to "standardize" not too long ago on `easymock`. It should be easy to replace `mockito` usage with `easymock` where applicable. For reference here is the jira that did that: https://issues.apache.org/jira/browse/KNOX-1479

Reply:
Moved to EasyMock to maintain code consistency.